server/sonar-server/src/main/java/org/sonar/server/computation/period/Period.java

```diff
@@ -22,19 +22,22 @@
 
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
-import org.apache.commons.lang.builder.ReflectionToStringBuilder;
+import javax.annotation.concurrent.Immutable;
 
+@Immutable
 public class Period {
```

Comment:
A internal class of Period do you mean ? Is it worth it just to remove one parameter ?